### PR TITLE
Added validate file for Read and uploadcontent

### PR DESCRIPTION
**Pull request by neha0305verma**

### CODE REVIEW

File: kp_service_test/src/test/resources/templates/content/v3/upload/testUploadResourceEcmlWithTwinAnimationAudioSpritesImgSprites/validate.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      "os": [
+        "All"
+      ],
+      "visibility": "Default",
+      "consumerId": "@ignore@",
+      "mediaType": "content",
+      "osId": "org.ekstep.quiz.app",
+      "languageCode": "en",
+      "version": 2,
+      "versionKey": "@ignore@",
```
Comment (krgauraw) [inline kp_service_test/src/test/resources/templates/content/v3/upload/testUploadResourceEcmlWithTwinAnimationAudioSpritesImgSprites/validate.json]: Please use isNumber function for all numeric fields.

File: kp_service_test/src/test/resources/templates/content/v3/read/testReadContentWithEditMode/response.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      ],
+      "visibility": "Default",
+      "consumerId": "@ignore@",
+      "mediaType": "content",
+      "osId": "org.ekstep.quiz.app",
+      "lastPublishedBy": "KP_FT_PUBLISHER",
+      "languageCode": "en",
+      "version": "@isNumber()@",
+      "pkgVersion": "@isNumber()@",
+      "versionKey": "@ignore@",
```
Comment (krgauraw) [inline kp_service_test/src/test/resources/templates/content/v3/read/testReadContentWithEditMode/response.json]: Please use isNumber() function . Please check in all files for numeric fields

Comment (neha0305verma) [reply kp_service_test/src/test/resources/templates/content/v3/read/testReadContentWithEditMode/response.json]: Fixed

File: kp_service_test/src/test/resources/templates/content/v3/upload/testUploadResourceEcmlWithTwinAnimationAudioSpritesImgSprites/validate.json (hunk 1 above)
Comment (neha0305verma) [reply kp_service_test/src/test/resources/templates/content/v3/upload/testUploadResourceEcmlWithTwinAnimationAudioSpritesImgSprites/validate.json]: Fixed

**Pull request merged**